specs/spec_template.md

```diff
@@ -0,0 +1,132 @@
+
+    For example, this section is a place to explain why you're adding this
+    API rather than modifying an existing API.
+
+    If you're writing a "converged" API add links into docs.microsoft.com
+    for the existing Win32 or WinRT APIs that are being converged.
+-->
+
+<!-- TEMPLATE
+    For example, this is a place to provide a brief explanation of some dependent
+    area, just explanation enough to understand this new API, rather than telling
+    the reader "go read 100 pages of background information posted at ...". 
+-->
+
+# Description
+
+<!-- TEMPLATE
+    Use this section to provide a brief description of the feature.
+    For an example, see the introduction to the PasswordBox control
+    (http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box).
+-->
+
+# Examples
+
+<!-- TEMPLATE
+
+    Use this section to explain the features of the API, showing
+    example code with each description. The general format is:
+
+    ## FirstFeatureName
+
+    Feature explanation text goes here, including why an app would use it, how it
+    replaces or supplements existing functionality.
+
+    ```c#
+    void SampleMethod() {
+        var show = new AnExampleOf();
+        show.SomeMembers = AndWhyItMight(be, interesting)
+    }
+    ```
+
+    ## SecondFeatureName
+
+    Feature explanation text goes here, including why an app would use it, how it
+    replaces or supplements existing functionality.
+
+    ```c#
+    void SampleMethod() {
+        var show = new AnExampleOf();
+        show.SomeMembers = AndWhyItMight(be, interesting)
+    }
+    ```
+
+    Code samples should be in C# and/or C++/WinRT.
+
+    As an example of this section, see the Examples section for the PasswordBox
+    control (https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). 
+-->
+
+# Remarks
+
+<!-- TEMPLATE
+
+    Explanation and guidance that doesn't fit into the Examples section.
+
+    APIs should only throw exceptions in exceptional conditions; basically,
+    only when there's a bug in the caller, such as argument exception.  But if for some
+    reason it's necessary for a caller to catch an exception from an API, call that
+    out with an explanation either here or in the Examples
+-->
+
+# API Notes
+
+<!-- TEMPLATE
+
+    Option 1: Give a one or two line description of each API (type and member),
+        or at least the ones that aren't obvious from their name. These
+        descriptions are what show up in IntelliSense. For properties, specify
+        the default value of the property if it isn't the type's default (for
+        example an int-typed property that doesn't default to zero.) 
+        
+    Option 2: Put these descriptions in the below API Details section,
+        with a "///" comment above the member or type. 
+-->
+
+# API Details
+
+<!-- TEMPLATE
+
+    The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) 
```

Comment:
Maybe give a hint to look at other completed API reviews for examples? Or recommend that they sketch the API with C# and then use WinMDIdl to reverse engineer it? Though I think that tool is still producing old IDL not IDL3. :/

Reply:
WinMDIDL still produces MIDLRT.  Ideally we'd get a WinMD->MIDL3 projection. Hey @BenJKuhn ...